cli_client/python/timesketch_cli_client/commands/sketch_test.py

```diff
@@ -35,11 +36,296 @@ def setUp(self):
     def test_list_sketches(self):
         """Test to list sketches."""
         runner = CliRunner()
+        # Ensure output format is 'text' for this test.
```

Comment:
fwiw, I triied to run the unit tests locally here and they failed, so this PR also fixed them